src/eyepop-render-2d/index.ts

```diff
@@ -15,7 +15,7 @@ import { RenderText, RenderTextOptions } from './render-text';
 
 export interface Renderer extends RenderTarget
 {
-    draw(p: Prediction): void
+    draw(p: Prediction, color?:string): void
```

Comment:
Typescript has a type ` @types/color` - should we use that instead here and in styles?

Reply:
Probably yes but will come back to this